lua/cfc_prop_restoration/core/sv/sv_init.lua

```diff
@@ -0,0 +1,102 @@
+local restoration_file_name = "props_backup.txt"
+local recent_disconnects = recent_disconnects or {}
+local prop_data = prop_data or {}
```

Comment:
What if we stored each user's prop data in separate files? That could reduce the I/O burden of loading them all at once, and instead only load them when needed.

Reply:
That sounds a bit unnecessary either way we're still going to be loading a ton of data...

Reply:
i would think the saving would be more of an issue
how would the server handle json encoding and saving every entity on the server in a single tick every minute

Reply:
Well it allows users to like.. save their data at will without having to re-dupe the entire map. Or allows us to do _something_ with a specific users data without modifying the entire structure

Reply:
Either way, we're going to have to go through all of the player's props to get a individual's props.